SCClassLibrary/Common/Control/Server.sc

```diff
@@ -405,11 +412,10 @@ Server {
 		};
 		clientID = val;
 		this.newAllocators;
-		if (statusWatcher.notNil and: { this.serverRunning }) {
-			this.initTree;
-		};
```

Comment:
why were these lines removed?

Reply:
we dont need initTree here anymore, it happens now at the end of 
Server:prHandleClientLoginInfoFromServer, where it should be.